database/dbService.js

```diff
@@ -15,19 +15,19 @@ module.exports = {
             var sql = queries.insertUser;
             con.query(sql, [user.id, user.gender, user.name.givenName, user.name.familyName, user.emails[0].value, placesLived, user.phoneNumber]
             , function (err, result) {
+                con.release();                                                
                 if (err) throw err;
-                callback(result.insertId);
-                con.release();                                
+                return callback(result.insertId);
             });
         });
     },
 
-    insertActivityLog: function(ID, firstName, lastName, action) {
+    insertActivityLog: function(ID, action) {
```

Comment:
I forgot to get rid of the first and last name parameters. Those no longer go into the activity log.